frontend/src/container/NotificationWall.jsx

```diff
@@ -0,0 +1,124 @@
+  ListItemWrapper,
+  PopinFixedHeader,
+  TracimComponent
+} from 'tracim_frontend_lib'
+
+export class NotificationWall extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      isNotificationWallOpen: true,
+      hasMoreNotifications: true
+    }
+  }
+
+  async componentDidMount () {
+    const { props } = this
+
+    const fetchGetNotificationWall = await props.dispatch(getNotificationList(props.user.userId))
+    switch (fetchGetNotificationWall.status) {
+      case 200:
+        props.dispatch(setNotificationList(fetchGetNotificationWall.json))
+        break
+      default:
+        this.sendGlobalFlashMessage(props.t('Error while loading notification list'))
+    }
+  }
+
+  handleClickBtnClose = notificationId => {
+    const { props } = this
+    const notification = props.notificationList.find(notification => notification.id === notificationId)
+    props.dispatch(updateNotification({ ...notification, read: true }))
+    this.setState({ isNotificationWallOpen: false })
+  }
+
+  handleClickSeeMore = () => { }
+
+  getNotificationTypeText = notification => {
+    const { props } = this
+    switch (notification.type) {
+      case 'content.created':
+        return props.t(' created ')
+      case 'content.modified':
+        return props.t(' updated a new version of ')
+      case 'status.modified':
+        return props.t(' updated the status of ')
+      case 'member.created':
+        return props.t(' added you ')
+    }
+  }
+
+  render () {
+    const { props, state } = this
+
+    if (!state.isNotificationWallOpen) return null
+
+    return (
+      <div className='notification'>
+        <PopinFixedHeader
+          customClass='notification'
+          faIcon='bell-o'
+          rawTitle={props.t('Notifications')}
+          componentTitle={<div>{props.t('Notifications')}</div>}
+          onClickCloseBtn={this.handleClickBtnClose}
+        />
+
+        <div className='notification__list'>
+          {props.notificationList && props.notificationList.map((notification, i) => {
+            return (
+              <ListItemWrapper
+                isLast={i === props.notificationList.length - 1}
+                read={false}
+                id={notification.id}
+                key={notification.id}
+              >
+                <Link
+                  to={notification.url}
+                  onClick={() => this.handleClickBtnClose(notification.id)}
+                  className={
+                    classnames('notification__list__item', { itemRead: notification.read })
+                  }
+                  key={notification.id}
+                >
+                  <i className={`notification__list__item__icon fa ${notification.icon}`} />
+                  <div className='notification__list__item__text'>
+                    {notification.author} {this.getNotificationTypeText(notification)}
+                    {notification.content ? <u>{notification.content}</u> : ''}
```

Comment:
@Skylsmoi 

What is important is to avoid mixing the content with the presentation, and that is why <b> is not recommended... is most cases. [It still has its uses and has been "undeprecated" in HTML5, with a new meaning](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b#Usage_notes). <span class="bold"> has the exact same issues as <b>, I'd even say it is worst, and <b> is always better than <span class="bold">. <strong> is the right tag to mark really important things semantically (and <em> for things that should be empathized). By the way, [<i> should be used instead of <em> in some use cases too](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i#Usage_notes)`.

As for underlining, it's use is discouraged by many experts on typography for readability reasons. <u> was deprecated in HTML 4 but then restored in HTML5 with a new meaning: ["mark text as having some form of non-textual annotation applied"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u).

Classes should not be used to mark styles. They should be used to give the "type" of something, that will then be styled accordingly. Life is not perfect and at times we don't have a choice.

I think this is a really important discussion we should have.

Reply:
You are correct, things are a bit overcomplicated.

And, in that particular case, your are saying to use neither of the proposed solution.

I'd say since we don't rely much on html semantic because these pages aren't parsed by any analytics robots (google and such), let's keep things simple.

My choice for simplicity is to use generic classes

Reply:
If your choice is simplicity: `<b>` is much simpler than `<span class="bold">`.

But I really think we should start a reflection on how to handle these things in Tracim. They matter for accessibility.

Moreover, I don't want to have to abide by rules for the sake of them. I want to know why I should. And I really don't see why I should write `<span class="bold">` instead of `<b>`, especially when I think the latter is better than the former and when I think that using generic classes like bold is actually a bad practice.

Reply:
@raphj @Skylsmoi I agree that `<b>` is simpler than `<span class="bold">`, but I also think that is good have classes for elements that we want a specific highlight. `<span class="documentTitle__highlight">` is too verbose for you?

Reply:
Perfect for me

Reply:
Changed